website/docs/podman/podman-remote.md

```diff
@@ -115,6 +139,12 @@ $ podman system connection default my-remote-machine
 $ podman ps
 ```
 
+:::note
+
+You can also use the `--connection` argument to only target the connection you want. (E.g. `podman --connection=my-remote-machine ps`)
```

Comment:
You can also use the `--connection` argument to target only the connection you want, for example, `podman --connection=my-remote-machine ps`)